canopeum_backend/canopeum_backend/models.py

```diff
@@ -218,6 +234,10 @@ def add_supported_specie_by_id(self, pk: int):
         tree_type = Treetype.objects.get(pk=pk)
         return BatchSupportedSpecies.objects.create(tree_type=tree_type, batch=self)
 
+    def plant_count(self):
+        batch_species = BatchSpecies.objects.filter(batch=self)
+        return reduce(lambda x, y: x + y.quantity, batch_species, 0)
```

Comment:
A reduce+add is just a sum ;)
```suggestion
        return sum(specie.quantity for specie in batch_species)
```